src/Fixer/Phpdoc/PhpdocSeparationFixer.php

```diff
@@ -126,8 +179,8 @@ private function fixAnnotations(DocBlock $doc): void
                 break;
             }
 
-            if (true === $next->getTag()->valid()) {
-                if (TagComparator::shouldBeTogether($annotation->getTag(), $next->getTag())) {
+            if (!$this->standardTagsOnly || $next->getTag()->valid()) {
```

Comment:
As far as I understand, the rule previously processed tags that were followed by a standard tag. So it made the following change:
```diff
 @foo
+
 @deprecated
```
bu left the following unchanged:
```diff
 @deprecated
 @foo
```
which is a bit weird.

I would drop the condition entirely in favor of considering that any tag (no matter standard or not) that is not part of the same group as the current tag should be separated from it. WDYT?

Reply:
Personally, I don't know what would be the most suitable approach. 
Maybe @jrmajor or @keradus should provide their opinions on this.

For now, I've changed condition so that preceding and following tags are processed whether they are standard PSR tags or not.

Reply:
I would also be in favor of dropping this flag, not sure how big of a BC break that would be though.

Reply:
IMO this is a bug, so dropping the condition is OK.

Reply:
The ``psr_standard_tags_only`` option and the condition are dropped!